WorldAround.UI/src/app/components/authentication/login/login.component.ts

```diff
@@ -0,0 +1,60 @@
+import { AfterViewInit, Component, ElementRef, OnDestroy, ViewChild, ViewEncapsulation } from '@angular/core';
+import { Router } from '@angular/router';
+import { LoginModel } from 'src/models/login';
+import { AuthorizationService } from 'src/services/authorization.service';
+import { ToastrService } from 'ngx-toastr';
+import { MatDialog, MatDialogRef } from '@angular/material/dialog';
+import { SignupComponent } from '../signup/signup.component';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss', '../authentication.scss']
+})
+export class LoginComponent implements OnDestroy, AfterViewInit {
+
+  @ViewChild('logUserName') userName: ElementRef;
+
+  public loginModel: LoginModel = new LoginModel();
```

Comment:
is public by default

Reply:
I suppose it will be much clear to specify the public